src/main/webapp/resources/js/components/ncbi/ExportUploadStateTag/NcbiUploadStateTag.tsx

```diff
@@ -26,7 +27,7 @@ const ExportUploadStateTag = ({
       {tagStates[state].text}
     </Tag>
   ) : (
-    <Tag className="t-upload-status" color={TagColor.ORANGE}>
+    <Tag className="t-upload-status" color={"orange"}>
```

Comment:
Add the orange color options to app.css and make use of it from there

Reply:
This isn't a true CSS value here so CSS variables will not work in this case.   If you go in and inspect the type definition for this it is: 
```
TagProps.color?: LiteralUnion<"magenta" | "red" | "volcano" | "orange" | "gold" | "lime" | "green" | "cyan" | "blue" | "geekblue" | "purple" | ..., string> | undefined
```
This is why I am supplying it with the string value "orange"

Reply:
Sounds good